examples/python_sdk/cancel_create_experiment.py

```diff
@@ -0,0 +1,27 @@
+from determined.common.experimental.determined import Determined
+
+
+def cancel_experiment(det: Determined) -> None:
+    exp1 = det.get_experiment(experiment_id=1)
+    print("Cancelling experimet id: {}".format(exp1.id))
+    exp1.cancel()
+
+
+def create_experiment_wait(det: Determined) -> None:
+    exp2 = det.create_experiment(
+        exp_config="../tutorials/mnist_pytorch/const.yaml",
+        model_dir="../tutorials/mnist_pytorch",
+    )
+    exp2.activate()
+    exp2.wait_till_complete()
+    print("Experiment {} {} is complete".format(exp2.config["description"], exp2.id))
+
+
+def main():
+    det = Determined(master="http://localhost:8080", user="determined")
+    cancel_experiment(det)
```

Comment:
Not as part of this PR, but looking at this example this seems like a case where we will really want to support a singleton pattern in addition to allowing users to pass around a `Determined()` object explicitly, sort of how the built-in `random` module lets you create an RNG explicitly (`Random()`) but it also [creates a singleton instance](https://github.com/python/cpython/blob/main/Lib/random.py#L831) of it and exposes of that singleton's methods as functions in the module.

Something to think about for the future, I think.